frontend/src/queries/useTokenQuery.ts

```diff
@@ -26,7 +26,7 @@ transaction {
 }
 event {
   __typename
-  ...on CisBurnEvent {
+  ...on Cis2BurnEvent {
```

Comment:
Can we avoid the breaking change of the name here?
So far we are backwards compatible.